src/config/pci-passthrough.md

```diff
@@ -0,0 +1,405 @@
+# PCI Passthrough
+
+PCI pass-through allows you to pass through an extra graphics card in your
+system with little to no performance drawback. This allows you to run
+applications that require GPU acceleration (CAD programs, video games, etc.) in
+your virtual machine on operating systems like Windows 10 with almost bare metal
+performance.
+
```

Comment:
Suggest something along the lines of: "PCI passthrough allows PCI devices to appear to be attached directly to the guest operating system in a virtual machine, with only minimal impact on performance. PCI passthrough can be thus used to improve the performance of VM applications which require GPU acceleration."

- Explains what "pass[ing] through" is.
- Avoids using pronouns such as "you" and "I".
- Has a less "marketing-speak" tone.

Reply:
I agree fixed.